src/dispatch/static/dispatch/src/incident/ReportReceiptResources.vue

```diff
@@ -0,0 +1,218 @@
+                <v-list-item-subtitle>Incident Commander</v-list-item-subtitle>
+              </v-list-item-content>
+              <v-row align="center" justify="end">
+                <v-btn
+                  v-if="commander.individual.phone"
+                  :href="'tel:' + commander.individual.phone"
+                  icon
+                  class="mr-1"
+                >
+                  <v-icon> mdi-phone </v-icon>
+                </v-btn>
+                <span class="mr-1"></span>
+                <v-btn :href="'mailto:' + commander.individual.email" icon class="mr-1">
+                  <v-icon> mdi-email </v-icon>
+                </v-btn>
+              </v-row>
+            </v-list-item>
+          </v-card-actions>
+        </v-card>
+      </v-col>
+      <v-col cols="12">
+        <v-card outlined elevation="0">
+          <v-list-item two-line>
+            <v-list-item-content>
+              <v-list-item-title class="text-h6"> Incident Resources </v-list-item-title>
+            </v-list-item-content>
+          </v-list-item>
+          <span v-for="pluginInstance in activeResourcePlugins" :key="pluginInstance.id">
+            <span v-if="pluginInstance.plugin.type === 'document'">
+              <span v-if="resourceData('documents').length">
+                <v-list-item
+                  v-for="document in resourceData('documents')"
+                  :key="document.resource_id"
+                  :href="document.weblink"
+                  target="_blank"
+                >
+                  <v-list-item-content>
+                    <v-list-item-title>{{ document.resource_type | deslug }}</v-list-item-title>
+                    <v-list-item-subtitle>{{ document.description }}</v-list-item-subtitle>
+                  </v-list-item-content>
+                  <v-list-item-action>
+                    <v-list-item-icon>
+                      <v-icon>open_in_new</v-icon>
+                    </v-list-item-icon>
+                  </v-list-item-action>
+                </v-list-item>
+                <v-divider />
+              </span>
+              <span v-else>
+                <v-list-item>
+                  <v-list-item-content>
+                    <v-list-item-title>Creating documents... </v-list-item-title>
+                    <v-progress-linear indeterminate />
+                  </v-list-item-content>
+                </v-list-item>
+              </span>
+            </span>
+            <span v-else>
+              <v-list-item
+                v-if="resourceData(pluginInstance.plugin.type)"
+                target="_blank"
+                :href="resourceData(pluginInstance.plugin.type).weblink"
+              >
+                <v-list-item-content>
+                  <v-list-item-title>{{
+                    pluginInstance.plugin.type | capitalize
+                  }}</v-list-item-title>
+                  <v-list-item-subtitle>{{
+                    pluginInstance.plugin.description
```

Comment:
I think showing the plugin type name and description instead of the resource name and description like before might be confusing for users.

Reply:
This is technically new functionality and won't affect the current submissions, but I agree they should be the same, afaik before we had one off strings that weren't derived from anywhere. This was my attempt as generalizing them. We can either use these strings, add new ones or take the hit and make it so each reasource is rendered individually.

Reply:
I think we should be consistent across the app and showing plugin info instead of resource info is confusing. When I look at the screenshot and I see the plugin title and description under incident resources it makes me think I created a plugin not a resource.

Reply:
I understand, but we have to choose how we accomplish this, our options are 1) Use these strings 2) Create new strings 3) Render each resource individually. It sounds like you prefer option 3?

Reply:
Yeah, I prefer # 3 as it's more clear for the user and consistent with what we already have.